src/modules/course/dto/update-lecture.dto.ts

```diff
@@ -24,4 +24,13 @@ export class UpdateLectureDto {
   @IsNotEmpty()
   @IsString()
   time: string;
+
+  /**
+   * The url video
+   * @example "https://codeforcause.org/video/1"
+   */
+  @IsNotEmpty()
+  @IsString()
+  @IsUrl()
+  lectureVideoUrl: string;
```

Comment:
This is update endpoint, I mean any field could remain unchanged at instance, for example while creating a lecture name has misspelled so only name need to be changed and not other properties, so don't impose isNotEmpty an all properties, instead add isOPtional.

Reply:
ok

Reply:
@Abhishek-kumar09 fixed